docs/en/essentials/getting-started.md

```diff
@@ -64,6 +64,27 @@ const app = new Vue({
 // Now the app has started!
 ```
 
+By injecting the router, we get access to it as `this.$router` as well as the current route as `this.$route` inside of any component:
+
+```js
+// Home.vue
+export default {
+  computed: {
+    params () {
+      // We will see what params are shortly
+      return this.$route.params
+    }
+  },
+  methods: {
+    someMethod () {
+      // Redirecting to another route
+      this.$router.push('/redirect')
+    }
```

Comment:
Again, a more real-world example could be helpful, so that readers can begin imagining when they might use `this.$router`. For example:

``` js
goBack () {
  window.history.length > 1
    ? this.$router.go(-1)
    : this.$router.push('/')
}
```